angular/core/components/thread_page/activity_card/new_comment_item_controller.coffee

```diff
@@ -1,5 +1,8 @@
 angular.module('loomioApp').controller 'NewCommentItemController', ($scope, $rootScope, $translate, Records, Session, ModalService, EditCommentForm, DeleteCommentForm, AbilityService, TranslationService, RevisionHistoryModal) ->
   $scope.comment = Records.comments.find($scope.event.eventable.id)
+  $scope.commentCollapsed = true
+  $scope.commentTooLong = $scope.comment.cookedBody().length > 600
```

Comment:
@robguthrie, do you have a sense of the best way to implement a specifically height-based or length-based solution?